### ISSUE TITLE
Formatter parenthesizes and expands multiple-assigment values

### ISSUE THREAD

**Issue by cnpryer** [astral-sh/ruff#7053]
I actually really like this result. Reporting anyway. I thought this was reported already, I but couldn't find any issues.

Line-length: 88

Black (23.7.0):
```py
def some_fn() -> None:
    df_aaaaaaa, df_aaaaaaa = res["aaaaaaaaaaaaaaaaa"].rename("aaa"), res[
        "aaaaaaaaaaaaaaaaa"
    ].rename("aaa")
```

Ruff (0.0.287):
```py
def some_fn() -> None:
    df_aaaaaaa, df_aaaaaaa = (
        res["aaaaaaaaaaaaaaaaa"].rename("aaa"),
        res["aaaaaaaaaaaaaaaaa"].rename("aaa"),
    )
```

**Comment by MichaReiser** [comment]
This matches [Black's preview style](https://black.vercel.app/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4AELAIZdAD2IimZxl1N_WlbvK5V_r8p8RWVQQnw9iapsVWAURY3SxClurVnV7aXA1aSAFyJDV_6KbhdEOAF6iWHwQpsW8LqvZgg0uA8_AllGpgDXu-99fTtWZaXXli_pe1jrTl-Pusr6e6XNQgFQsZYiQf0UEuVCiygH0IBiAkjiBxZoJLpZsqRswVAAAAAA6vtt-_240qIAAaIBjAIAAGWQ6g6xxGf7AgAAAAAEWVo=) and IMO, is more readable overall. 

How frequent was this change in your code base? I recommend leaving as is if it was a rare change.

**Comment by charliermarsh** [comment]
In general, I find it reasonable not to fix deviations that already exist in Black's preview style, though of course depends on how common they are, how hard they are to fix, etc.

**Comment by cnpryer** [comment]
This was the most frequently altered pattern. It's common throughout my test suite.

Very much prefer this though.

**Comment by cnpryer** [comment]
I'm going to update the snippet I provided for clarity. Looks like these are all series-based tests.

**Comment by MichaReiser** [comment]
Okay this has a different reason than I initially thought and you correctly called it out:

> I'm going to update the snippet I provided for clarity. Looks like these are all series-based tests.

The issue is that ruff prefers parenthesizing the tuple in assignment positions rather than breaking them left to right (the default is right to left). I personally find this much easier to read. I struggled a lot scanning what the assignment targets and what the values are. 

This is probably related to another Ruff deviation in relation with Tuples:

Ruff is inserting parentheses for tuple's in more places than Black. I wasn't aware that this is also happening in assignments, but we're deviating in expression statements because I and @konstin believe that it overall improves readability. For example a common incorrect use of tuples in django ([PR](https://github.com/django/django/pull/17181)) was

```python
label_for_field("nonexistent", Article, form=ArticleForm()),
```

Which clearly is an incidental use of tuples. Ruff formats this as 

```
(label_for_field("nonexistent", Article, form=ArticleForm()),)
```

This makes the tuple very prominent, making you aware of the potential mistake (but Ruff removes the parentheses once there are at least two values).

I'm interested to hear @konstin opinion on this issue.

**Comment by MichaReiser** [comment]
> This matches [Black's preview style](https://black.vercel.app/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4AELAIZdAD2IimZxl1N_WlbvK5V_r8p8RWVQQnw9iapsVWAURY3SxClurVnV7aXA1aSAFyJDV_6KbhdEOAF6iWHwQpsW8LqvZgg0uA8_AllGpgDXu-99fTtWZaXXli_pe1jrTl-Pusr6e6XNQgFQsZYiQf0UEuVCiygH0IBiAkjiBxZoJLpZsqRswVAAAAAA6vtt-_240qIAAaIBjAIAAGWQ6g6xxGf7AgAAAAAEWVo=) and IMO, is more readable overall.
> 
> How frequent was this change in your code base? I recommend leaving as is if it was a rare change.

I need to revise this statement. Black's preview style isn't changing the output. The output only changes when the tuple on the right is parenthesized in the source, in which case Black preserves the parentheses.

**Comment by konstin** [comment]
> I'm interested to hear @konstin opinion on this issue.

For the given example, i strongly prefer the ruff style over the black stable style, i find the black stable style confusing wrt to seeing that the RHS is a tuple expression. I'm a bit worried though that there are cases where the deviations are unintended.

**Comment by MichaReiser** [comment]
Related to #7317 Closing to track the discussion there.
